eogrow/utils/testing.py

```diff
@@ -168,10 +168,14 @@ def _calculate_tiff_stats(tiff_filename: str, config: StatCalcConfig) -> JsonDic
 def _calculate_vector_stats(gdf: gpd.GeoDataFrame, config: StatCalcConfig) -> JsonDict:
     """Calculates statistics over a vector GeoDataFrame"""
 
-    def _rounder(x: float, y: float) -> tuple[float, float]:
+    def _rounder(coords: tuple[float, float]) -> tuple[float, float]:
+        x, y = coords
         return _prepare_value(x, np.float64), _prepare_value(y, np.float64)
```

Comment:
if this `_rounder` function is meant only for vector stats anyway, how about extending it to work also on `Point` objects, since intuitively it doesn't really make a difference

```suggestion
    def _rounder(coords: tuple[float, float] | Point) -> tuple[float, float]:
        x, y = (coords.x, coords.y) if isinstance(coords, Point) else coords
        return _prepare_value(x, np.float64), _prepare_value(y, np.float64)
```

since this then makes it look better down the line (will post suggestions)